### PR TITLE
 Add authorization policy for accessing a reading room resource

### PR DESCRIPTION
**Pull request by hackmastera**
refs #3119

### CODE REVIEW

File: app/models/ability.rb [hunk 1]
```diff
@@ -182,7 +182,21 @@ def valkyrie_test_read(obj)
   end
 
   def group_readable?(obj)
-    (user_groups & obj.read_groups).any?
+    groups = (user_groups & obj.read_groups)
+    if groups == ["reading_room"]
```
Comment (tpendragon) [inline app/models/ability.rb]: Change this to use the constant?

**Comment by hackmastera** [comment]
nope; I didn't have the radio button so I figured that will be another PR.

**Comment by hackmastera** [comment]
This is fixed up; ready for re-review.

**Comment by hackmastera** [comment]
@tpendragon well and now this pr has the radio button.
Screen Shot 2019-07-12 at 4 11 58 PM

**Pull request merged**